sparse/impl/KokkosSparse_twostage_gauss_seidel_impl.hpp

```diff
@@ -985,8 +986,11 @@ class TwostageGaussSeidel {
                 Kokkos::subview(localZ, Kokkos::ALL(), range_type(j, j + 1));
             single_vector_view_t Rj(localRj.data(), num_rows);
```

Comment:
@masterleinad, @dalg24: Is there a way to specify the execution space instance here?

For example,
https://github.com/kokkos/kokkos-kernels/blob/8497ad264582f07b50c6325bf5418a409f6fa68d/sparse/src/KokkosSparse_gauss_seidel_handle.hpp#L761-L763

Reply:
@e10harvey Subviewing and unmanged view construction don't do any API calls or kernel launches that could be queued up on a stream, so you don't have to worry about them. They just manipulate the Kokkos::View (pointer/strides/extents) structures on host.